resource_booking/tests/common.py

```diff
@@ -42,7 +42,7 @@ def create_test_data(obj):
                 "name": "Fridays",
                 "dayofweek": "4",
                 "hour_from": 0,
-                "hour_to": 23.99,
+                "hour_to": 24,
```

Comment:
This should go in the same direction as the migration script, but I don't understand why it's needed.

Reply:
This change checks https://github.com/OCA/calendar/pull/120#discussion_r1519952482